tests/integration/usage_statistics/test_usage_statistics_messages.py

```diff
@@ -62,8 +62,6 @@ def test_generate_messages_with_defaults():
         "event_time": "2020-08-04T22:50:58.837Z",
         "data_context_id": TEST_DATA_CONTEXT_ID,
         "data_context_instance_id": TEST_DATA_CONTEXT_INSTANCE_ID,
-        "mac_address": TEST_MAC_ADDRESS,
-        "oss_id": TEST_OSS_ID,
```

Comment:
We shouldn't update our old examples - I've deleted all these new identifier additions and then added a few test cases with the persistent values (making sure to utilize a modern version of GX, a recent event time, etc to model a real-life event)